docs/README.md

```diff
@@ -23,7 +23,7 @@ npm install --save bitski
 Alternatively you can add this script tag to your app’s `<head>`:
 
 ```html
-<script src="https://cdn.jsdelivr.net/npm/bitski@0.1.9/lib/bitski.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/bitski@0.2.0/lib/bitski.js"></script>
```

Comment:
Do we need to include more detail here now that web3 is unbundled?

Reply:
I don't know if anyone really needs CDN stuff.